.evergreen/env-var-utils.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+# Usage:
+#   to_absolute "./relative/path/to/file"
+#   to_absolute "./relative/path/to/dir"
+#   to_absolute "/absolute/path/to/file"
+#   to_absolute "/absolute/path/to/dir"
+#
+# Uses dirname, basename, and pwd only. Unix-style paths only!
+to_absolute() (
+  declare path="${1:?"to_absolute requires a path to convert"}"
+
+  declare dir_part base_part
+  dir_part="$(dirname "${path}")" || return
+  base_part="$(basename "${path}")" || return
+
+  declare res
+
+  if [[ -d "${path}" ]]; then
+    cd "${path}" || return
+    res="$(pwd)" || return
+  else
+    res="$(to_absolute "${dir_part}")" || return
+    [[ "${res: -1:1}" == "/" ]] || res+="/" || return
+    [[ "${base_part}" == "~" ]] || res+="${base_part}" || return
+  fi
```

Comment:
This is probably a lot more efficient than my `abspath` function in libmongocrypt, which walks up the entire path without the `cd $dir; pwd` trick (it is annoyingly slow on Windows).